server.py

```diff
@@ -97,7 +97,8 @@ def decrypt():
     except Exception as e:
         return server_error(e)
     else:
-        logger.info("Decrypted received data")
+        bound_logger = logger.bind(tx_id=decrypted_json.get("tx_id"))
+        bound_logger.info("Decrypted received data")
```

Comment:
I know you've only copied the existing string, but should `Decrypted ` actually be `sdx-decrypt`?